src/FakeAsyncClient.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Http\Client\Utils;
+
+use Http\Client\HttpAsyncClient;
+use Http\Client\HttpClient;
+use Http\Client\Exception;
+use Http\Client\Utils\Promise\FulfilledPromise;
+use Http\Client\Utils\Promise\RejectedPromise;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Transform a HttpClient into a HttpAsyncClient
+ *
+ * @author Joel Wurtz <joel.wurtz@gmail.com>
+ */
+class FakeAsyncClient implements HttpAsyncClient
+{
+    /** @var \Http\Client\HttpClient  */
```

Comment:
ah, you already do. just Httpclient is good then. we have class member phpdoc on multiline, so this should be
```
/**
 * @var HttpClient
 */
```